components/sponsored_benefits/app/views/sponsored_benefits/organizations/plan_design_proposals/plan_reviews/estimated_employee_cost_details.html.erb

```diff
@@ -1,61 +1,63 @@
 <div class="container">
   <h2><b>Benefit Package Set Up</b></h2>
   <h3><b>Selected Reference Plan</b></h3>
-  <div class="panel col-md-12 col-xs-12">
-    <div class="panel-body">
-      <div class="row">
-        <div class="col-xs-8">
-          <h4 class="heading-text mt-0"><b><%= @reference_plan.name %></b></h4>
-        </div>
-        <div class="col-xs-4">
+  <% unless @show_dental_plan_details%>
```

Comment:
this whole block can be refactored. based on kind, you can set reference plan instance variable either with health or dental (something like below)
```
@reference_plan = health? ? @benefit_group.reference_plan : @benefit_group.dental_reference_plan
```

Reply:
Thank you, this simplified a lot.